### PR TITLE
Added option to transform records before sending to ES

### PR DESCRIPTION
**Pull request by cdelgadob**
Pending update README, but ready for testing

### CODE REVIEW

File: .git-commit-template.txt [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+#    fix      (bug fix)
+#    refactor (refactoring production code)
+#    style    (formatting, missing semi colons, etc; no code change)
+#    docs     (changes to documentation)
+#    test     (adding or refactoring tests; no production code change)
+#    chore    (updating grunt tasks etc; no production code change)
+# --------------------
+# Remember to
+#    Capitalize the subject line
+#    Use the imperative mood in the subject line
```
Comment (matrus2) [inline .git-commit-template.txt]: You submit this rule and at the same time your commit message looks in the following way:

> feat(transformation): Added posibility to add a transformation functi…

>…on to modify the object.

Would be really nice if after review fixes you could leave only one commit with all your changes and proper commit message.

Comment (matrus2) [reply .git-commit-template.txt]: But I like this file :)

File: package-lock.json [hunk 2]
```diff
@@ -10,7 +10,7 @@
       "integrity": "sha512-OfC2uemaknXr87bdLUkWog7nYuliM9Ij5HUcajsVcMCpQrcLmtxRbVFTIqmcSkSeYRBFBRxs2FiUqFJDLdiebA==",
```
Comment (matrus2) [inline package-lock.json]: Not sure what is happening here. Pls revert changes for this file.

**Comment by cdelgadob** [comment]
Closing this pull request as I had to recreate the repo

**Pull request closed without merging**